tests/Event/EmitterTest.php

```diff
@@ -139,7 +139,7 @@ public function testRemoveListener(): void
         );
 
         $ee->emit('foo');
-        self::assertFalse($result);
+        self::assertFalse($result); /* @phpstan-ignore-line */
```

Comment:
I think you should ignore the error by phpstan config with a error message pattern instead of adding it on every line

Reply:
yes, that will be nicer for this repeated stuff.

Note to future readers: phpstan thinks that `$result` is always `false` and so complains that this assert can never fail. Actually it is testing that the event really did not happen, and phpstan thinks "of course nothing happened to change the value of `$result`", so the assert is a waste of space. But that is a bit over-confident of phpstan.

Reply:
done

Reply:
I think we are talking about this phpstan issue: https://github.com/phpstan/phpstan/issues/2018

Reply:
seems this one will be fixed in the next phpstan release, see https://github.com/phpstan/phpstan/issues/2018#issuecomment-1491464767